compiler/src/org.graalvm.compiler.core.llvm/src/org/graalvm/compiler/core/llvm/LLVMUtils.java

```diff
@@ -61,6 +62,16 @@
         public static final int NODE = 3;
     }
 
+    public abstract static class TargetSpecific {
```

Comment:
Can you put documentation comments here? This will be the API for future architectures.

Reply:
I was thinking about putting this in when we actually support ARM, instead of having bits of ARM support hanging around which might make it look like it works with ARM when it doesn't

Reply:
done